src/index.js

```diff
@@ -30,30 +30,19 @@ const Container = ({ options }) =>
 const onfidoRender = (options, el, merge) =>
   render( <Container options={options}/>, el, merge)
 
-const stripOneCapture = ({blob, documentType, onfidoId, side}) => {
-  const capture = {id: onfidoId, blob}
-  if (documentType) capture.documentType = documentType
-  if (side) capture.side = side
-  return capture
-}
-
-const stripCapturesHash = captures => mapValues(captures,
-  capture => capture ? stripOneCapture(capture) : null)
+const mapCaptures = captures => mapValues(captures, capture => capture)
 
-const getCaptures = () => mapKeys(stripCapturesHash(events.getCaptures()), key => key + 'Capture')
+const getCapturesKeys = () => mapKeys(mapCaptures(events.getCaptures()), key => key + 'Capture')
 
 function bindEvents (options) {
   const eventListenersMap = {
```

Comment:
There's only one event now so I don't think we need this map anymore

Reply:
If we plan to add other events in the future I'd rather leave like it is. Otherwise I can get rid of all the bind and unbind events logic and create a function that only listen to the complete event. What do you guys think?

Reply:
I am always a fan of getting rid of redundant code. It will be easy to add it back later if we need it